lib/generate-suggestions.ts

```diff
@@ -0,0 +1,96 @@
+import fs from "fs-extra";
+import glob from "glob";
+import { parse } from "@babel/parser";
+import traverse from "@babel/traverse";
+
+import { fetchComponents } from "./http/fetchComponents";
+
+async function generateSuggestions() {
+  const components = await fetchComponents();
+  const results: {
+    [compApiId: string]: FindResults;
+  } = {};
+
+  for (const [compApiId, component] of Object.entries(components)) {
+    if (!results[compApiId]) {
+      results[compApiId] = [];
+    }
+    const result = await findTextInJSXFiles(".", component.text);
+    results[compApiId] = [...results[compApiId], ...result];
+  }
+  console.log(JSON.stringify(results, null, 2));
+}
+
+interface Occurence {
+  lineNumber: number;
+  preview: string;
+}
+
+type FindResults = { file: string; occurrences: Occurence[] }[];
+
+async function findTextInJSXFiles(
+  path: string,
+  searchString: string
+): Promise<FindResults> {
+  const result: { file: string; occurrences: Occurence[] }[] = [];
+  const files = glob.sync(`${path}/**/*.+(jsx|tsx)`);
```

Comment:
I think we should probably implicitly ignore `node_modules` since every JavaScript codebase is going to have it

Reply:
![image](https://user-images.githubusercontent.com/15795866/230448206-bcf53fbb-0fe0-4079-bd3a-21260ba1a7ae.png)